### PR TITLE
Pract.07: Add 01 | Polina Doncheva

### PR DESCRIPTION
**Pull request by polinaDoncheva**
Polina Doncheva
6 9MI0600223

### CODE REVIEW

File: SI/Sem.07/Pract.07/References/pract07zad2.cpp [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+
+#include <iostream>
+using namespace std;
+
+void toUpper(char& letter) 
+{
+	int difference = 'a' - 'A';
+	letter -= difference;
```
Comment (GeorgiTerziev02) [inline SI/Sem.07/Pract.07/References/pract07zad2.cpp:9]: one line

File: SI/Sem.07/Pract.07/References/pract07zad2.cpp [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+void toLower(char& letter)
+{
+	int difference = 'a' - 'A';
+	letter += difference;
```
Comment (GeorgiTerziev02) [inline SI/Sem.07/Pract.07/References/pract07zad2.cpp:15]: one line

**Review by GeorgiTerziev02 — APPROVED** [review]

File: SI/Sem.07/Pract.07/References/pract07.cpp [hunk 3]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (Justsvetoslavov) [inline SI/Sem.07/Pract.07/References/pract07.cpp:1]: Empty line 1 and 2.
Move the code 2 lines upwards.

**Review by Justsvetoslavov — CHANGES_REQUESTED** [review]
Remove unnecessary empty lines.
Address the comments.

**Pull request merged**